driver/Dockerfile

```diff
@@ -14,8 +14,19 @@ RUN apt-get update && apt-get install -y git wget bash bzip2 zip && \
     mkdir $MAMBA_ROOT_PREFIX && \
     micromamba install -y --prefix $MAMBA_ROOT_PREFIX \
         -c cpg -c bioconda -c conda-forge \
-        hail=$HAIL_VERSION cpg-utils analysis-runner sample-metadata=3.5.0 bokeh selenium phantomjs statsmodels \
-        r-base=4.1.1 r-essentials r-tidyverse r-argparser && \
+        hail=$HAIL_VERSION \
+        analysis-runner \
+        bokeh \
+        cpg-utils \
+        phantomjs \
+        r-argparser && \
+        r-base=4.1.1 \
+        r-essentials \
+        r-tidyverse \
+        sample-metadata=3.5.0 \
+        selenium \
+        skopeo \
+        statsmodels \
```

Comment:
```suggestion
        statsmodels; \
```

Reply:
final import no longer followed by && or a termimation

Reply:
Thanks a lot for catching this! I've changed it to `&&` to make sure we stop immediately in case of an error.